.identity/04_github_environment.tf

```diff
@@ -0,0 +1,32 @@
+resource "github_repository_environment" "prod_opex_ci" {
+  environment = "${var.env}-ci"
+  repository  = local.github.repository
+}
+
+resource "github_repository_environment" "prod_opex_cd" {
+  environment = "${var.env}-cd"
```

Comment:
Sorry I didn't notice it before, but I'd like to change the name `opex-${var.env}-ci` as done in `io-sign`

Reply:
done in [14ee036](https://github.com/pagopa/io-web-profile-backend/pull/32/commits/14ee0365658a58dfc25fe0c9fbb78d25f4fb2503)